bridge/ffi/src/bindings.rs

```diff
@@ -210,6 +210,9 @@ pub enum Action {
 
 		/// connection details
 		connection: Connection,
+
+		/// optional key to import
+		dsnp_keys: *mut DsnpKeys,
```

Comment:
I'm not sure about this. Trying to represent a ffi compatible `Option<DsnpKeys>`